.github/workflows/test.yml

```diff
@@ -83,8 +83,10 @@ jobs:
       - name: Run Go tests
         run: go test -covermode atomic -coverprofile coverage.txt $(go list ./... | grep -v third_party/)
       - name: Workaround buggy Codecov OIDC auth
-        if: github.event_name == 'push'
         run: |
+          # only set CODECOV_TOKEN if OIDC token is available
+          [ -z $ACTIONS_ID_TOKEN_REQUEST_TOKEN ] && exit 0
```

Comment:
could we do `if: ${{ env.ACTIONS_ID_TOKEN_REQUEST_TOKEN != '' }}`

Reply:
I can try it out, I guess it's a little more visible to users

Reply:
actually, it doesn't work: I remember now that environment variables are not available in the task conditional.